css/interface.css

```diff
@@ -20,6 +20,22 @@ body {
   padding: 0;
 }
 
+.help-icon {
+  font-size: 16px;
+  margin-left: 8px;
+  color: #333;
+  opacity: 0.5;
+}
+
+header {
```

Comment:
1. Why is is necessary to add a class rather than defining styles for `header p`?
2. What is the point of adding a class if you're splitting your css declarations between HTML tag and actual class? Please move this declaration to the `.header-wrapper` you just added.
3. Please rename the class as it's not a wrapper but rather the container itself, e.g. use `.header`.

Reply:
cc @tonytlwu ^

Reply:
It does look like `header` and `header p` will do the job. No classes are necessary.